sdcm/nemesis.py

```diff
@@ -1337,56 +1333,59 @@ def _check_pod_stats(node):
         assert final_target_pod_stats
         assert final_target_pod_stats.get("restart_count") == origin_target_pod_stats.get("restart_count")
 
-    def disrupt_terminate_and_replace_node_kubernetes(self):  # pylint: disable=invalid-name
-        for node_terminate_method_name in self._get_kubernetes_node_break_methods():
-            self.set_target_node()
-            InfoEvent(f'OperatorNodeTerminateAndReplace ({node_terminate_method_name}) {self.target_node}').publish()
-            if not self._is_it_on_kubernetes():
-                raise UnsupportedNemesis('OperatorNodeTerminateAndReplace is supported only on kubernetes')
+    def disrupt_drain_kubernetes_node_then_replace_scylla_node(self):
+        self._disrupt_kubernetes_then_replace_scylla_node('drain_k8s_node')
 
-            self._disrupt_terminate_and_replace_node_kubernetes(self.target_node, node_terminate_method_name)
+    def disrupt_terminate_kubernetes_host_then_replace_scylla_node(self):
+        raise UnsupportedNemesis("https://github.com/scylladb/scylla-operator/issues/1124")
```

Comment:
now that we know it's working as we expect in serverless, let's not skip it for serverless